src/App/Document.cpp

```diff
@@ -262,8 +262,9 @@ bool Document::redo(const int id)
     return false;
 }
 
-void Document::addOrRemovePropertyOfObject(TransactionalObject* obj,
-                                           const Property* prop, const bool add)
+void Document::changePropertyOfObject(TransactionalObject* obj,
+                                      const Property* prop,
+                                      std::function<void()> changeFunc)

```

Comment:
As linter says. Alternatively you can change it to template to not use `std::function` wrapper if not needed.